integration-tests/deployment/ccip/propose.go

```diff
@@ -237,3 +211,39 @@ func BuildProposalMetadata(state CCIPOnChainState, chains []uint64) (map[mcms.Ch
 	}
 	return tlAddressMap, metaDataPerChain, nil
 }
+
+// Given batches of operations, we build the metadata and timelock addresses of those opartions
+// We then return a proposal that can be executed and signed
+func BuildProposalFromBatches(state CCIPOnChainState, batches []timelock.BatchChainOperation, description string, minDelay string) (*timelock.MCMSWithTimelockProposal, error) {
+	if len(batches) == 0 {
+		return nil, fmt.Errorf("no operations in batch")
+	}
+
+	if minDelay != "" {
+		minDelay = "0s"
+	}
```

Comment:
I think you mean:

```suggestion
	if minDelay == "" {
		minDelay = "0s"
	}
```

[`time.Duration`](https://pkg.go.dev/time#Duration) is the standard way to represent durations in Go, and is represented by a int64 under the hood. In that scenario, you can remove this check, because 0s is the default value of a duration.